### ISSUE TITLE
Could not find a valid loopback address after upgrading to nodejs 18.0

### ISSUE THREAD

**Issue by jmnovak50** [homebridge/HAP-NodeJS#942]
### Analysis

When upgrading to Node 18.0 noticing errors

### Expected Behavior

No errors

### Steps To Reproduce

Upgrade to node 18.0

### Logs

```markdown
Error: Could not find a valid loopback address on the platform!
    at findLoopbackAddress (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/net-utils.ts:38:11)
    at getOSLoopbackAddressIfAvailable (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/net-utils.ts:60:50)
    at new HAPConnection (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/eventedhttp.ts:361:103)
    at EventedHTTPServer.onConnection (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/eventedhttp.ts:201:24)
    at Server.emit (node:events:527:28)
    at TCP.onconnection (node:net:1632:8)
```

### Configuration

```json
Node 18.0
Homebridge 1.4.0
```

### Environment

- OS: Debian GNU/Linux Bullseye (11) 
- Homebridge: 1.4
- Node: 18.0
- npm:8.7.0

### Process Supervisor

hb-service

**Comment by NorthernMan54** [comment]
Duplicate of https://github.com/homebridge/homebridge/issues/3114 and https://github.com/homebridge/.github/issues/6

Thank you for reporting this, we will investigate.

Please note that Node.js v18 will not be officially supported until 2022-10-25.

**Comment by jmnovak50** [comment]
Thanks for investigating @NorthernMan54!  Not a major one but it looks like it’s already on the radar. 🙂

**Comment by oznu** [comment]
Fixed as of v0.10.2-beta.3

A release will come shortly.
